element-hq/element-x-android
# Render blocked user data (behind a disabled feature flag)

## Type of change

- [x] Other : customer request

## Content


When feature flag `ShowBlockedUsersDetails` is enabled, the application will render display name and avatar of blocked users. This can be useful for professional environment. Disabled by default on EXA.

## Motivation and context


Customer request.

## Screenshots / GIFs


See recorded ones.

## Tests



- Enable the flag
- Block a user who have a display name and an avatar
- Navigate to the blocked users screen
- Observe that user avatar and display name is rendered.

## Tested devices

- OS version(s):

## Checklist
